src/connector/Cargo.toml

```diff
@@ -82,8 +82,9 @@ mysql_common = { version = "0.31", default-features = false, features = [
 ] }
 nexmark = { version = "0.2", features = ["serde"] }
 num-bigint = "0.4"
-opendal = "0.44"
+opendal = "0.44.2"
 parking_lot = "0.12"
+parquet = "50.0.0"
```

Comment:
Any specific reason why we pin the exact version instead of just the major version here for these two crates?